package.json

```diff
@@ -9,7 +9,7 @@
     "test:e2e": "cypress run",
     "build": "webpack",
     "start": "webpack-serve",
-    "lint": "eslint src"
+    "lint": "eslint src tests cypress"
```

Comment:
In order to test Vue SFCs you'd need to add the `--ext` flag to lint `.vue` files.